src/components/common/Loader.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import '../common/styles/Loader.scss';
+export default function Loader() {
+  return (
+    <div class="sk-chase">
+      <div class="sk-chase-dot"></div>
+      <div class="sk-chase-dot"></div>
+      <div class="sk-chase-dot"></div>
+      <div class="sk-chase-dot"></div>
+      <div class="sk-chase-dot"></div>
+      <div class="sk-chase-dot"></div>
+    </div>
```

Comment:
Just googled this chase dot because i was curious as to why there's 6 of the same dot. Very interesting add! UX/UI buff